otter-blocks.php

```diff
@@ -49,6 +49,18 @@ function ( $products ) {
 	}
 );
 
+add_filter(
+	'themeisle_sdk_compatibilities/' . basename( OTTER_BLOCKS_PATH ),
+	function ( $compatibilities ) {
+		$compatibilities['OtterBlocksPRO'] = array(
+			'basefile'  => defined( 'OTTER_PRO_BASEFILE' ) ? OTTER_PRO_BASEFILE : '',
+			'required'  => '2.0',
+			'tested_up' => OTTER_BLOCKS_VERSION,
```

Comment:
here I think it should be the PRO version, not the free one, since the metadata is about the PRO plugin

Reply:
We're just using the version of the plugin as both pro/free are on the same version. Using Pro constant here, or the Free in Pro, will just break it as it will just take whatever version of the plugin is available as tested_up, even if the other plugin is a year old.

Reply:
makes sense, if they are in sync is good.